src/components/Privacy/createComponent.js

```diff
@@ -39,10 +43,10 @@ export default ({
     commands: {
       setConsent: {
         optionsValidator: validateSetConsentOptions,
-        run: options => {
+        run: ({ preferences }) => {
```

Comment:
I will run this name by @dancingcactus & Karthik and get back to you.

Reply:
@Aaronius @jonsnyder I ran this by Karthik and he believes `preferences` conflict with marketing concepts; so we think maybe `consent` instead?

Reply:
Sure.